Formula/faas-cli.rb

```diff
@@ -0,0 +1,35 @@
+class FaasCli < Formula
+  desc "CLI for templating and/or deploying FaaS functions"
+  homepage "http://docs.get-faas.com/"
+  url "https://github.com/alexellis/faas-cli/archive/0.3.tar.gz"
+  sha256 "d4db7ddb0ea1dadf469c3da9893b004874d19570d09457e8cf86e5aef0df00f5"
+
+  depends_on "go" => :build
+  depends_on "docker" => :build
```

Comment:
```
Josephs-MacBook-Pro:Formula joe$ ag 'depends_on "docker"'
boot2docker.rb
23:  depends_on "docker" => :recommended

docker-compose.rb
22:  depends_on "docker" => :recommended

gitlab-ci-multi-runner.rb
19:  depends_on "docker" => :recommended

minimesos.rb
9:  depends_on "docker"

monax.rb
16:  depends_on "docker"
```
Just anticipating that someone will probably ask for it to be possible to use their own docker.